app/src/main/java/com/duckduckgo/app/onboarding/ui/page/WelcomePage.kt

```diff
@@ -71,8 +70,7 @@ class WelcomePage : OnboardingPageFragment(R.layout.content_onboarding_welcome)
     private var typingAnimation: ViewPropertyAnimatorCompat? = null
     private var welcomeAnimationFinished = false
 
-    // we use a BroadcastChannel because we don't want to emit the last value upon subscription
```

Comment:
This comment seems important; it says we don't want to emit the last value... but will `replay = 1` do just that?

Reply:
I added the comment back, changed it to:
```
// we use replay = 0 because we don't want to emit the last value upon subscription
private val events = MutableSharedFlow<WelcomePageView.Event>(replay = 0, extraBufferCapacity = 1)
```

Reply:
Looks good